parcels/examples/tutorial_FieldLists.ipynb

```diff
@@ -24,6 +24,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "%matplotlib inline\n",
```

Comment:
Do you know what does that line?

Reply:
If we don't have this line, then the first plot of the notebook is not shown. Something strange with matplotlib initialisation...